.github/workflows/release.yml

```diff
@@ -18,38 +17,27 @@ jobs:
       uses: docker/setup-qemu-action@v2
       with:
         image: tonistiigi/binfmt:latest
```

Comment:
```suggestion
```

Reply:
Sorry, should have added a comment. I just wanted to point out (as elsewhere) that this is the default (as per: https://github.com/docker/setup-qemu-action#inputs) which I believe we could (should?) omit